mmcv/cnn/bricks/wrappers.py

```diff
@@ -34,7 +37,8 @@ def backward(ctx, grad):
 class Conv2d(nn.Conv2d):
 
     def forward(self, x):
-        if x.numel() == 0 and TORCH_VERSION <= (1, 4):
+        if x.numel() == 0 and (TORCH_VERSION == 'parrots' or TORCH_VERSION <=
```

Comment:
I suggest defining a `obsolete_torch_version(version_threshold)` function and change this line to `if x.numel() == 0 and obsolete_torch_version((1, 4)):`